code/drasil-example/dblpend/lib/Drasil/DblPend/Assumptions.hs

```diff
@@ -39,11 +39,11 @@ cartSysDesc :: Sentence
 cartSysDesc = atStartNP (a_ cartesian) `S.is` (S "used" !.)
 
 cartSysRDesc :: Sentence
-cartSysRDesc = atStartNP (the cartesian) `S.is` S "right-handed where" +:+ 
+cartSysRDesc = atStartNP (the cartesian) `S.is` S "right-handed" +:+ S "where" +:+ 
```

Comment:
This change is not an improvement, please remove. If it used a combinator, it would have been, but this is just splitting English text needlessly.